client/src/main/java/io/pravega/client/admin/ReaderGroupManager.java

```diff
@@ -71,8 +72,9 @@ public static ReaderGroupManager withScope(String scope, ClientConfig clientConf
      * may block.
      * @param groupName The name of the group to be created.
      * @param config The configuration for the new ReaderGroup.
+     * @return True if ReaderGroup is created with the provided config.
```

Comment:
"is created" read to me as "was created" IE in response to this call. So I missed the meaning of this change the first time. We should either clarify the docs or change the behavior.

I would prefer the later, returning false vs true for something that is idempotent makes sense. But if it already exists and the passed config is different that isn't a case for idempotence. That seems like an exceptional case to me.

Reply:
I have improved the API signature based on this feedback.